_posts/2020-01-26-newpipe-0.18.2-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube-Extractor"
```

Comment:
That dash shouldn't be there, gramatically speaking.

Reply:
me grammar great.